Lesson3/ConsoleAppExtensions.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Project.Interface;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Project.ConsoleApp
+{
+    public static class ConsoleAppExtensions
+    {
+        const string _json = "-json";
+        const string _xml = "-xml";
+
+        /// <summary>
+        /// SortNullsToBottomAndReplaceWithStringValue
+        /// </summary>
+        /// <param name="items">string collection of items to sort</param>
+        /// <param name="nullReplacement">string literal to replace nulls with. Example: "No Value"/param>
+        /// <returns>an array of sorted items with nulls at the bottom and replaced with a string literal</returns>
+        public static IEnumerable<string> SortNullsToBottomAndReplaceWithStringValue(this IEnumerable<string> items, string nullReplacement)
```

Comment:
`Observation`: This method seems unneeded.  It just combines two methods that are already simple to use.  It's name is even a combination of what you would get if you just chained the methods together.